app/Console/Commands/ResetAdmin.php

```diff
@@ -1,72 +0,0 @@
-<?php
```

Comment:
I am wondering if it would not be better to keep that file but have it print "use php artisan lychee:update_user  <username> <password> instead".

Still I am wondering, what happens if you forgot the admin username & password. :|